pkg/skaffold/initializer/init.go

```diff
@@ -372,9 +374,15 @@ func generateSkaffoldConfig(k Initializer, buildConfigPairs []builderImagePair)
 	// if the user doesn't have any k8s yamls, generate one for each dockerfile
 	logrus.Info("generating skaffold config")
 
+	name, err := suggestConfigName()
+	if err != nil {
+		return nil, errors.Wrap(err, "generating default config name")
```

Comment:
should this be an error ? since its a optional field, we should log this as error and continue

Reply:
Yes, I agree.